assets/src/dashboard/templates/README.md

```diff
@@ -0,0 +1,26 @@
+# Template Generation From Raw Story JSON
+
+`templates/raw` is a dumping ground for raw story json output from the editor, which is then interpretted as a template.
```

Comment:
Nit, 'interpretted' should be 'interpreted'

Reply:
lol yea I cannot spell for the life of me 🤦